chromium_src/chrome/browser/extensions/manifest_v2_experiment_manager.cc

```diff
@@ -0,0 +1,33 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/extensions/manifest_v2_experiment_manager.h"
+
+#include "brave/browser/extensions/manifest_v2/features.h"
```

Comment:
Can you use this forward-declaration technique to eliminate the need to include a brave file here? https://github.com/brave/brave-core/pull/30984/files#diff-07d7235ead46632a187c2fb39037a32eaad6676580d9c07c8aa7553ef518d5cbR8

Otherwise, we would need to add a dep and it gets problematic quickly. Sorry I didn't notice this the first time.

Reply:
But chrome/browser/extensions already depends on brave/browser/extensions, or I missed something ?